packages/react-components/react-components/etc/react-components.api.md

```diff
@@ -1106,6 +1107,8 @@ export { OnVisibleChangeData }
 
 export { OpenPopoverEvents }
 
+export { OverridesContextValue_unstable }
```

Comment:
I don't think that we want expose it, especially taking into account that it's reachable via `FluentProviderProps['unstable_overrides']`.